### ISSUE TITLE
Option to add a new question to a question group

### ISSUE THREAD

**Issue by sethshoultes** [eventespresso/event-espresso-core#350]
Codebase ticket:

We currently have a similar feature in the question group editor (https://www.screencast.com/t/I3tGd7Ota). 

Here is what I am thinking. We could add a meta box in the question editor for new and existing questions, where the user can choose to add the current question to a group. 

**For MVP:**

- I think it would be fine to limit the question group selections to one from this page.
- If there are a lot of question groups (over 15), then we could add a lazy load or link to load more.
- I don't think we need to worry about drag and drop ordering features for the question groups.
- I don't think we need to add a question group from here. maybe add a link to the question groups tab in the meta box description?

**Mock-up:**
![questions-with-question-groups](https://user-images.githubusercontent.com/235315/39260747-a74ad5d4-4877-11e8-952e-b92b99e07218.jpg)

**Comment by nerrad** [comment]
@sethshoultes I wonder if this is something that should be held off and done as part of a general refactor of the questions/question groups interface?  I think @tn3rb  had something already started somewhere that would improve things (maybe you could chime in with your thoughts on this issue too Brent?)

**Comment by nerrad** [comment]
Another consideration if you _do_ decide to go ahead with this anyways Seth, is how will this function with regards to how the questions are ordered in the group?  Questions are sortable within the Question Group interface, so if a question is added to a question group via the question group editor, what order should the new question get in that group?  Just append to the list of questions in it?

**Comment by sethshoultes** [comment]
Yes, I would just append the new question to the end of the selected group, then add a link/icon that points each question group editor.

**Comment by sethshoultes** [comment]
@tn3rb mentioned you were working on a refactor of the question/question group interface. Can you comment on that?

**Comment by tn3rb** [comment]
I think now that React + Gutenberg is coming down the pipe, the refactor that I started (at least the JS portion) is kinda DOA.
There's some elements to what I started, such as updating the DB and models so that questions and question groups more closely resemble our form system inputs and form sections may still be applicable, but even that may need to be refactored to work better with React.

So that said, I think we can go ahead with this change since it is pretty straight forward.
